### ISSUE TITLE
Error: Variables must be length 1 or 767. Problem variables: 'text'

### ISSUE THREAD

**Issue by Boel** [sahirbhatnagar/manhattanly#18]
I am using v 0.2.0

The highlight code from the vignette works, but when I try to highlight variants in my data set I get:
```{r}
 manhattanly(subset(tmp,chr %in% 6), chr='chr',gene = "gene",snp='SNP', bp='pos',annotation1='OR',highlight=the6)
Error: Variables must be length 1 or 767.
Problem variables: 'text'
all(the6 %in% tmp[,'SNP'])
[1] TRUE
```
Also, when I try to highlight only one variant all points become highlighted (green). 
```{r}
manhattanly(subset(tmp,chr %in% 6), chr='chr',snp = "SNP", gene = "gene",bp='pos',annotation1='OR',highlight='A_77_30018741_S')
```

**Comment by sahirbhatnagar** [comment]
This happens because only one chromosome is being plotted. If we take the example from the vignette:

This works as expected:
```R
manhattanly(subset(HapMap, CHR %in% 4:7), snp = "SNP", gene = "GENE", highlight = significantSNP)
```
But this fails as it highlights all points:
```R
manhattanly(subset(HapMap, CHR %in% 4), snp = "SNP", gene = "GENE", highlight = significantSNP)
```
Not sure what's going on here. I need to look further into this as it might be due to the changes in plotly.

**Comment by sahirbhatnagar** [comment]
Fixed via #23
